internal/compiler-bridge/src/main/scala/xsbt/CallbackGlobal.scala

```diff
@@ -171,10 +171,12 @@ sealed class ZincCompiler(settings: Settings, dreporter: DelegatingReporter, out
 
         case None => // The compiler outputs class files in a classes directory (the default)
           // This lookup could be improved if a hint where to look is given.
-          outputDirs
-            .map(_.resolve(classFilePath))
-            .find(Files.exists(_))
-            .map(Compat.plainNioFile(_))
+          if (classFilePath.contains("<")) None
```

Comment:
Okay as a defensive backstop, but we should require the inputs to this method to be valid classfile paths. My fix to `fullName` and `processDependency` _should_ get us there.

Reply:
Yea. For this PR I just wanted to get AppVeyor going, but I'd be happy to remove this part if/when it's no longer needed.